src/Security/Member.php

```diff
@@ -76,6 +76,7 @@ class Member extends DataObject
         'Password' => 'Varchar(160)',
         'AutoLoginHash' => 'Varchar(160)', // Used to auto-login the user on password reset
         'AutoLoginExpired' => 'Datetime',
+        'AutoLoginTempHash' => 'Varchar(64)', // Temporary hash used for auto-login
```

Comment:
```suggestion
        'AutoLoginTempHash' => 'Varchar(64)', // Temporary hash used for auto-login to prevent leaking AutoLoginHash on redirect
```
Just distinguishing it further from `AutoLoginHash` so it's clearer without referring to code in other files why we have two.

Reply:
Updated

I've also moved all the inline comments to above the field they're for because the updated inline comment was too long for the lines char limit